Doc/library/zipfile.rst

```diff
@@ -818,7 +818,18 @@ Instances have the following methods and attributes:
 .. attribute:: ZipInfo.date_time
 
    The time and date of the last modification to the archive member.  This is a
-   tuple of six values:
+   tuple of six values representing the "last mod file time" and "last mod file date"
```

Comment:
```suggestion
   tuple of six values representing the "last [modified] file time" and "last [modified] file date"
```
I realize you are using the text from APPNOTE.txt here, but I think we should expand the word to be a bit clearer.